privacy.tex

```diff
@@ -84,8 +84,8 @@ \section{Collection of personal information}
 
 \subsection{How is your personal information collected?}
 
-We typically collect personal information about members through the membership joining process directly from members. This is done both through our relationship with University of Southampton Students' Union, and through signups to our mailing list.We may sometimes collect additional information from third parties including the University of Southampton and the University of Southampton Students’ Union. 
-We may collect additional personal information in the course of member-related activities throughout the period of your membership with us. Particular examples of this are signing up to events, or registering for an account on one of our web services.
+We typically collect personal information about members through the membership joining process directly from members. This is done both through our relationship with University of Southampton Students' Union, and through signups to our mailing list. We may sometimes collect additional information from third parties including the University of Southampton and the University of Southampton Students’ Union.\\
+We may collect additional personal information in the course of member-related activities throughout the period of your membership with us. Particular examples of this are signing up to events, registering for an account on one of our web services, or authenticating to access our network.
```

Comment:
Remove this ", or authenticating to access our network"
And add a new bullet point
" - We collect data through our IDM (Identity Management Platform) sign up process. " Or something

Reply:
In \section{Use of personal information} add a clause "We use you data to confim identity to provide access to appreciate services and infrastruture"